firmware/src/sgx/src/untrusted/main.c

```diff
@@ -157,6 +162,11 @@ int main(int argc, char **argv) {
     unsigned int tx = 0;
 
     while (true) {
+        if (G_stop_requested) {
+            G_stop_requested = 0;
+            finalise_with(0);
+        }
+
```

Comment:
I would just exit the loop instead, and call `finalise_with(0);` instead of the unexpected loop exit log. No need to reset the flag either.

Reply:
Pushed a new commit, let me know how it looks